File: CPP/Tiket.cpp
#include <string>
using namespace std;

class Tiket {
private:
    string id;
    string namaFilm;
    string jenisTiket;
    int harga;

public:
    Tiket(string id, string namaFilm, string jenisTiket, int harga) {
        this->id = id;
        this->namaFilm = namaFilm;
        this->jenisTiket = jenisTiket;
        this->harga = harga;
    }

    string getId() {
        return id;
    }

    string getNamaFilm() {
        return namaFilm;
    }

    string getJenisTiket() {
        return jenisTiket;
    }

    int getHarga() {
        return harga;
    }

    void setNamaFilm(string namaFilm) {
        this->namaFilm = namaFilm;
    }

    void setJenisTiket(string jenisTiket) {
        this->jenisTiket = jenisTiket;
    }

    void setHarga(int harga) {
        this->harga = harga;
    }
};

File: Python/tiket.py
class Tiket:
    def __init__(self, id, namaFilm, jenisTiket, harga):
        self.__id = id
        self.__namaFilm = namaFilm
        self.__jenisTiket = jenisTiket
        self.__harga = harga

    def get_id(self): return self.__id
    def get_namaFilm(self): return self.__namaFilm
    def get_jenisTiket(self): return self.__jenisTiket
    def get_harga(self): return self.__harga

    def set_namaFilm(self, value): self.__namaFilm = value
    def set_jenisTiket(self, value): self.__jenisTiket = value
    def set_harga(self, value): self.__harga = value
